lib/actual_db_schema/patches/migration_context.rb

```diff
@@ -4,15 +4,22 @@ module ActualDbSchema
   module Patches
     # Add new command to roll back the phantom migrations
     module MigrationContext
-      def rollback_branches
+      def manually_rollback_branches
         migrations.reverse_each do |migration|
-          migrator = down_migrator_for(migration)
-          migrator.extend(ActualDbSchema::Patches::Migrator)
-          migrator.migrate
+          next unless status_up?(migration)
+
+          puts File.read(migration.filename)
+          prompt_user_for_migration(migration)
```

Comment:
WDYT if we change it a bit so that prompt user input is responsible only for getting the result from the user, e.g.:

```ruby
migrate(migration) if user_wants_rallback
```

`user_wants_rallback`  could return true if the answer is "y", and false otherwise.